pkg/reconciler/revision/resources/init.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"path"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	internalVolumeName      = "knative-internal"
+	internalVolumeMountPath = "/var/knative-internal"
+
+	podInfoVolumeName      = "pod-info"
+	podInfoVolumeMountPath = "/var/pod-info"
+	podInfoPodName         = "podname"
+	podInfoNamespace       = "namespace"
+)
+
+var (
+	initVolumeMounts = []corev1.VolumeMount{{
+		Name:      internalVolumeName,
+		MountPath: internalVolumeMountPath,
+	}, {
+		Name:      podInfoVolumeName,
+		MountPath: podInfoVolumeMountPath,
+	}}
+
+	// Create a symlink to enable fluentd log collection on the host
+	//   - From container POV: /var/knative-internal/{NAMESPACE}_{POD_NAME}_{USER_CONTAINER_NAME} -> ../knative-var-log
+	//   - From host POV: <POD_VOLUME_DIRECTORY>/knative-internal/{NAMESPACE}_{POD_NAME}_{USER_CONTAINER_NAME} -> ../knative-var-log
+	// This way, there is a folder on the host with the metadata required by fluentd for proper event enrichment (see 100-fluentd-configmap.yaml)
+	initArgs = []string{
+		"/bin/sh",
+		"-c",
+		"ln -s ../" + varLogVolumeName + " " +
```

Comment:
Could fmt.sprintf make this command more readable?

Reply:
Done.